### DISCUSSION TITLE
Why the text displayed by react-pdf is broken :(?

### DISCUSSION THREAD

**Discussion by octavian-regatun** [wojtekmaj/react-pdf#999] [Q&A]
Hello,

Anyone can help me fix this problem, or has any knowledge of why the text looks like this, please?

![Screenshot_1](https://user-images.githubusercontent.com/31423936/168466714-835f5c35-5790-4a5c-aea3-810bff8ec543.png)

**Comment by dprothero** [comment]
I'm seeing the same thing.
![image](https://user-images.githubusercontent.com/1897006/210031432-945e73b8-756b-4313-93d4-031e98b3eb5e.png)

As opposed to what it should look like:
![image](https://user-images.githubusercontent.com/1897006/210031511-79ff3a9e-f359-4f6b-b35b-46500e91e074.png)

**Reply by DanO-Gmail** [reply]
I'm getting the same thing. Only on chromium based browsers from what I can tell (ff is fine). @octavian-regatun @dprothero , did you have any luck figuring this out?

**Comment by wojtekmaj** [comment]
Check the latest version of React-PDF. This has been recently fixed in v6.2.2.
